src/main/java/org/folio/service/processing/ParallelFileChunkingProcessor.java

```diff
@@ -165,8 +165,7 @@ private void processFiles(JobProfileInfo jobProfile,
       boolean succeeded = ar.succeeded();
       LOGGER.info("Sending event to Kafka finished. ar.succeeded(): {} jobProfile: {}, eventType: {}",
         succeeded, jobProfile, eventType);
-      LOGGER.debug("Closing KafkaProducer jobProfile: {}", jobProfile);
-      producer.end(par -> LOGGER.debug("KafkaProducer has been closed jobProfile: {}", jobProfile));
+      producer.end(par -> producer.close());
       processFilePromise.handle(ar);
```

Comment:
Logs for starting/finishing closing producers written by native Kafka implementation, we dont need to duplicate them here